types/registries/nacos/v2/config.proto

```diff
@@ -0,0 +1,33 @@
+// Copyright The HTNN Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package types.registries.nacos.v2;
+
+import "google/protobuf/duration.proto";
+import "validate/validate.proto";
+
+option go_package = "mosn.io/htnn/types/registries/nacos/v2";
+
+message Config {
```

Comment:
How many differences between Nacos v1 & v2? If the Nacos v2 shares the most options with the v1, we can just add a new field to indicate if v2 or v1 is used.

Reply:
There are not many differences between Nacos v1 and v2 in terms of configuration options. Both versions share the same fields